src/blocks/shape-divider/index.php

```diff
@@ -51,11 +51,11 @@ function coblocks_render_coblocks_shape_divider_block( $attributes ) {
 		? "background-color: {$attributes['customBackgroundColor']};"
 		: false;
 
-	$is_vertically_flipped = isset( $attributes['verticalFlip'] )
+	$is_vertically_flipped = ( isset( $attributes['verticalFlip'] ) && true === $attributes['verticalFlip'] )
```

Comment:
We can simplify this with `! empty( $attributes['verticalFlip'] )`

Alternatively, we can swap the condition results and remove the negation:
```
empty( $attributes['verticalFlip'] ) ? false : 'is-vertically-flipped'
```

Reply:
Great suggestion.